opencti-platform/opencti-graphql/src/modules/attributes/internalObject-registrationAttributes.ts

```diff
@@ -109,7 +109,7 @@ const internalObjectsAttributes: { [k: string]: Array<AttributeDefinition> } = {
   ],
```

Comment:
Bookmarks field in User type is also an object [{id, type}] : 
{ name: 'bookmarks', type: 'json', mandatoryType: 'no', multiple: true, upsert: false },

Reply:
Is id,type but stored in string, so json type for me

Reply:
I don't see it stored as String, here is the mapping I have locally : 

```
"bookmarks": {
  "properties": {
    "id": {
      "type": "text",
          "fields": {
        "keyword": {
          "type": "keyword",
              "ignore_above": 512,
              "normalizer": "string_normalizer"
        }
      }
    },
    "type": {
      "type": "text",
          "fields": {
        "keyword": {
          "type": "keyword",
              "ignore_above": 512,
              "normalizer": "string_normalizer"
        }
      }
    }
  }
},
```

And here is an example where it is stored :  

```
"bookmarks": [
  {
    "id": "b446835d-66f5-4a46-b57b-de11de93a68a",
    "type": "Threat-Actor-Individual"
  },
  {
    "id": "fe16663c-6b94-4857-9640-e53573560f02",
    "type": "Intrusion-Set"
  },
  {
    "id": "1ee7de1c-c6bc-48e5-9766-4de1f2f59f8b",
    "type": "Threat-Actor-Group"
  }
]

```